### PR TITLE
add area limiter and mixing length parameters

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Adds some parameters that are needed for area fraction limiter in entrainment and detrainment. Also adds Smagorinsky coefficient and minimum mixing lenght.

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: src/parameters.toml [hunk 1]
```diff
@@ -2139,6 +2151,18 @@ value = 0.12
...
+value = 0.1
+type = "float"
+description = "Constant coefficient that scales the maximum area limiter term in detrainment. Parameter not described in the literature."
+
+[max_area_limiter_power]
+alias = "max_area_limiter_power"
+value = 10
+type = "float"
+description = "Constant coefficient for the exponent in the maximum area limiter term in detrainment. Parameter not described in the literature."
+
```
Comment (szy21) [inline src/parameters.toml:2165]: I can also not add these two parameters and use the ones we have [here](https://github.com/CliMA/CLIMAParameters.jl/blob/945eb72ea84cf5ad8231bcdd7715e8a57dd11e45/src/parameters.toml#L2250-L2260). Just the description is not accurate. What do you think is better?

Comment (trontrytel) [reply src/parameters.toml:2165]: Maybe we could:
1. Add new ones with the correct descriptions
2. Add the ones we need to delete to this garbage collecting issue: https://github.com/CliMA/CLIMAParameters.jl/issues/99
3. Promise to actually handle this issue before the next sprint ;) @nefrathenrici , @szy21  - What do you think?

**Review by trontrytel — APPROVED** [review]

**Comment by szy21** [comment]
@trontrytel I added two more parameters :)

**Pull request merged**